rebasehelper/cli.py

```diff
@@ -160,14 +160,17 @@ def run():
                 raise RebaseHelperError('Wrong format of --builder-options. It must be in the following form:'
                                         ' --builder-options="--desired-builder-option".')
             cli = CLI()
-            if cli.version:
+            if hasattr(cli, "version"):
```

Comment:
You obviously prefer double quotes, that's fine, but you should keep the quoting style consistent among code blocks.
With the exception of `"--builder-options"` (which should be fixed) every string in this method uses single quotes.